### ISSUE TITLE
bounding box of edges causes erroneous selection in visual editor

### ISSUE THREAD

**Issue by Mearman** [breadboard-ai/breadboard#2453]
see video:

https://github.com/breadboard-ai/breadboard/assets/1331872/52ebd0be-f088-4b89-b302-227521068213

**Comment by paullewis** [comment]
We currently use rect-rect intersection, which is on occasion a bit coarse. But to do poly-poly intersection is a decent bit of code that I'm not sure is warranted at this stage.

**Comment by Mearman** [comment]
yeah I guessed it wouldn't be an easy one unless it was already provided by Pixi
